### PR TITLE
Vehicle routing problem with fixed penalty time windows

### PR DESCRIPTION
**Pull request by owenlacey**
I couldn't find any examples like this online, so thought I'd create an example, as I've had this problem to solve recently.

This is essentially a vehicle routing problem with the following constraints:
- Each order has a time window
- Rather than enforcing the model conforming to the time windows, we'll penalise any lateness.
- This penalty with be fixed, as opposed to a progressively worse penalty depending on how late it is.

### CODE REVIEW

**Comment by owenlacey** [comment]
@googlebot I fixed it.

**Comment by owenlacey** [comment]
@googlebot I fixed it.

File: examples/contrib/vrptw_fixed_penalty.cs [hunk 1]
```diff
@@ -0,0 +1,169 @@
+// Copyright 2012 Hakan Kjellerstrand
```
Comment (Mizux) [inline examples/contrib/vrptw_fixed_penalty.cs:1]: Not sure of this copyright claims, should be this instead
`// Copyright 2021 owenlacey` (or both lines ?)
or whatever you want (e.g. your employer if any).

**Review by Mizux — APPROVED** [review]

**Comment by Mizux** [comment]
note: PR must be done against our dev branch which is master, stable branch is our default and release branch.

ps: Github force default branch (for cloning, and on github.com) to be also the default branch for PR -_-

File: examples/contrib/vrptw_fixed_penalty.cs (hunk 1 above)
Comment (owenlacey) [reply examples/contrib/vrptw_fixed_penalty.cs]: of course - sorry that's some lazy copy-pasting on my part

**Comment by stradivari96** [comment]
Also make sure you signed the CLA with the email that appears here:
https://github.com/google/or-tools/commit/e5d09b54aeb603acadab7d3da10ac9eafbf8926f.patch

**Comment by owenlacey** [comment]
Apologies, I'm going to abandon this because the commits were authored by an alternative (and incorrect account). I couldn't see a quick way around this so thought I'd redo the changes in another PR.

Other pr here: https://github.com/google/or-tools/pull/2358

**Pull request closed without merging**